static/templates/settings/display-settings.handlebars

```diff
@@ -37,7 +37,7 @@
                     {{/if}} />
                     <span></span>
                 </label>
-                <label for="left_side_userlist" class="inline-block">{{t "Display user list on left side in narrow windows." }}</label>
+                <label for="left_side_userlist" class="inline-block">{{t "User list on left in narrow windows" }}</label>
```

Comment:
I think this description could be better if we're editing it and thus making translations re-translate.  Maybe "User list on left sidebar in narrow windows"?